src/main/java/tutoraid/model/student/Lessons.java

```diff
@@ -7,8 +7,8 @@
 import tutoraid.commons.util.AppUtil;
```

Comment:
Is this the class that we might be changing to LessonNames?

Reply:
Yes